modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|`rhcos-46.82.202008071940-0`
+|`ami-04f6587e3cce5a66f`
+|`rhcos-45.82.202008010929-0`
+|`ami-062235cc7876c4ecb`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0a574de130fd8efc3`
+|`rhcos-49.84.202107191624-0`
+|`ami-0f175eaa2161dc2ed`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0ea57d1b0b4a20f12`
+|`rhcos-48.84.202105062123-0`
+|`ami-0e2ccf6b05ed3a339`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-08bce188c387a560e`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0015282afaac5c4b6`
+|`rhcos-45.81.202005200134-0`
+|`ami-020e22cc494ab6f27`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0d08c5f66ac8265e0`
+|`rhcos-48.84.202107202156-0`
+|`ami-0f7d388ea57cf6520`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0792d53f3a6136a14`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-00ce33bc6c6aeebf4`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-008e70e09e4a791d3`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0f9c9c300516d22c7`
+|`rhcos-46.82.202008181646-0`
+|`ami-0eba9f75078b5554a`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0eac1d2661a5f772a`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0aeff31e74d221f73`
+|`rhcos-47.83.202102090044-0`
+
+.19+|*eu-west-2*
+|`ami-0fea90cff84a2735e`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0743f240a3763b39c`
+|`rhcos-45.81.202005200134-0`
+|`ami-0928a5c22afde08e0`
+|`rhcos-46.82.202008071940-0`
+|`ami-09f4baef0e163b5d4`
+|`rhcos-48.84.202105062123-0`
+|`ami-0a4ceb29453ae76f7`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-02d46da64377e7e23`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-048fbd2974b114590`
+|`rhcos-49.84.202107191624-0`
+|`ami-0368f0af2f16e6fdb`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-03c005b7cfa8114cd`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-005c77358f657cf60`
+|`rhcos-46.82.202008181646-0`
+|`ami-034e60a30bf173ea5`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-00f8b34bbeefddddd`
+|`rhcos-45.82.202008010929-0`
+|`ami-07adb9340bf130666`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0b059d767817b443f`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-078a02880a7f28fb9`
+|`rhcos-48.84.202107202156-0`
+|`ami-0ded752197186c01e`
+|`rhcos-47.83.202102090044-0`
+|`ami-0387f83845eb1a32b`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0eeb9e8f6ede0099c`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-05c3f2ad7963594f3`
+|`rhcos-414.92.202305090606-0-x86_64`
+
+.19+|*eu-south-1*
+|`ami-07dc6682b8de235a8`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-02e6ff7df83e20271`
+|`rhcos-48.84.202107202156-0`
+|`ami-09885f805b18b7542`
+|`rhcos-49.84.202107191624-0`
+|`ami-06bc4f03cca2834f6`
+|`rhcos-48.84.202105062123-0`
+|`ami-06d5068b959daeef1`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-09b8d42580e13dfbb`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-037df250c060f1a85`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-06f41f191c93c2473`
+|`rhcos-47.83.202102090044-0`
+|`ami-05cb3152ba1bfea18`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?